src/runtime/create.ts

```diff
@@ -114,7 +114,7 @@ export function setupRuntime(notebook: StarboardNotebookElement): Runtime {
       notebook.performUpdate();
     },
 
-    runCell(id: string, focusNext: boolean, insertNewCell: boolean) {
+    runCell(id: string, focusNext?: "before" | "after", insertNewCell?: boolean) {
```

Comment:
Ah `focusNext` now implies `next` as in `after` time-wise. Kind of confusing (I can't think of a much better name though, `focusAfterRun` or just `focus` maybe?

Reply:
Okay, I renamed it to `focus` with `previous` and `next` as possible values. I hope that's fine :)